packages/@lwc/template-compiler/src/__tests__/config.spec.ts

```diff
@@ -44,7 +44,7 @@ describe('customRendererConfig normalization', () => {
                 ],
               },
               "enableDynamicComponents": false,
-              "enableLwcSpread": false,
+              "enableLwcSpread": true,
```

Comment:
This line can be removed right?

Reply:
Leaving it for now since removing it fails tests in lwc-platform.